fatiando/mesher.py

```diff
@@ -1720,3 +1720,433 @@ def keep(cell):
             return True
         return False
     return [c for c in cells if keep(c)]
+
+
+class TriaxialEllipsoid (GeometricElement):
+
+    """
+    Create a triaxial ellipsoid.
+
+    .. note:: The coordinate system used is x -> North, y -> East
+    and z -> Down
+
+    Parameters:
+
+    * x, y, z : float
+        The coordinates of the center of the ellipsoid
+    * a, b, c : float
+        Semi-axes of the ellipsoid (a>b>c)
```

Comment:
`a, b, c` are terrible variable names. It's really hard to know what they are just from the name and they mean different things for different ellipsoids. It would be better to use their proper names if they have them. If not, use `large_axis`, `median_axis`, `small_axis` or something along those lines.